common/reviews/api/rush-sdk.api.md

```diff
@@ -0,0 +1,58 @@
+## API Report File for "@rushstack/rush-sdk"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+// @public
+export interface ILoadSdkAsyncOptions {
+    abortSignal?: AbortSignal;
+    onNotifyEvent?: SdkNotifyEventCallback;
+    rushJsonSearchFolder?: string;
+}
+
+// @public
+export interface IProgressBarCallbackLogMessage {
+    message: string;
+    messageType: 'info' | 'debug';
+}
+
+// @public
+export interface ISdkCallbackEvent {
+    logMessage: IProgressBarCallbackLogMessage | undefined;
+    progressBarPercent: number | undefined;
+    state: SdkCallbackState;
+}
+
+// @public
+export namespace rushSdkProxy {
+    export function loadAsync(options?: ILoadSdkAsyncOptions): Promise<void>;
+}
+
+// @public
+export type SdkCallbackState =
+/**
+* The task has started and is still running.
+*/
```

Comment:
BTW these comments are a very interesting API Extractor bug. I will see if I can fix it in a separate PR.